crates/fdev/src/testing/multiple_process.rs

```diff
@@ -396,13 +396,12 @@ async fn child(
     child_config: &MultiProcessConfig,
 ) -> anyhow::Result<()> {
     let id = child_config.id.expect("id should be set for child process");
-    let data_dir = child_config
+    let _data_dir = child_config
```

Comment:
if we are not setting this, then we don't need it.
how are we setting where data is stored then though?

Reply:
I'm going to remove it. The data_dir variable was used to set this environment variable: std::env::set_var("FREENET_DATA_DIR", data_dir); Now this env var is not used.

Reply:
fixed